.github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml

```diff
@@ -49,15 +49,16 @@ concurrency:
 
 jobs:
   beam_Python_ValidatesContainer_Dataflow_ARM:
-    name: beam_Python_ValidatesContainer_Dataflow_ARM
+    name: beam_Python_ValidatesContainer_Dataflow_ARM (Run Python ValidatesContainer Dataflow ARM 3.x)
```

Comment:
Could we remove 3.x from the name and trigger commands?

Reply:
Followed what was done here: https://github.com/apache/beam/blob/afa4371b1ddaf017c817d9a833d3a6d04310236c/.github/workflows/beam_PreCommit_Python_Integration.yml#L54

Reply:
looks like we do need to specify matrix element. then, it's intentional. nvm.